### PR TITLE
test: Add tests for single account PCA

### PR DESCRIPTION
**Pull request by Peetee06**
This PR adds tests for Single Account PCA.

They are a prerequisite to ensure everything keeps working when making refactorings while adding browser support.

A future PR will add tests for Multiple Account PCA.

### CODE REVIEW

File: lib/src/utils/utils.dart [hunk 1]
```diff
@@ -16,26 +16,28 @@ final class Utils {
     IosConfig? iosConfig,
   }) async {
     final arguments = <String, dynamic>{};
-    if (Platform.isAndroid) {
```
[the remark this answers is not in the document]
Comment (nayanAubie) [reply lib/src/utils/utils.dart:19]: Yes @Peetee06 we need to change it to `defaultTargetPlatform`

**Comment by nayanAubie** [comment]
@Peetee06 I can not go ahead with these changes because these changes are new to me and I need to check in-depth for the web

**Comment by Peetee06** [comment]
@nayanAubie This PR only adds tests for the current functionality. It does not change any functionality, except for using the semantically equal `defaultTargetPlatform`. So you can safely go ahead with this PR as soon as you reviewed and approved the tests. 

As for the new files coming with PR #51 you are welcome to take any time you need. 👍

**Comment by nayanAubie** [comment]
@Peetee06 Hello Peter, you need to redirect your PRs to another branch, `feat/msal-js`. because we don't want any issues in the main as we are working on other features and in the upcoming days, we may need to release new versions before web support.

**Comment by Peetee06** [comment]
@nayanAubie ah, thanks for the hint! I redirected this branch to `feat/msal-js` now 👍

**Pull request merged**